compiler/damlc/tests/daml-test-files/Iou_template.EXPECTED.md

```diff
@@ -1,58 +1,58 @@
-# Module Iou_template
-
-
+# <a name="module-ioutemplate-98694"></a>Module Iou\_template
 
 ## Templates
 
-### Template `Iou`
-
-
-| Field        | Type/Description |
-| :----------- | :----------------
-| `issuer`     | `Party` |
-| `owner`      | `Party` |
-| `currency`   | `Text` |
-|              | only 3-letter symbols are allowed |
-| `amount`     | `Decimal` |
-|              | must be positive |
-| `regulators` | `[` `Party` `]` |
-|              | `regulators` may observe any use of the `Iou` |
+### <a name="template-ioutemplate-iou-32396"></a>Template Iou
 
+| Field      | Type/Description |
+| :--------- | :----------------
+| issuer     | Party |
```

Comment:
Did we remove the backticks here on purpose?

Reply:
Yes, to make it closer to Rst docs and to allow for links in the type.